templates/templated_email/order/confirm_order.email

```diff
@@ -0,0 +1,17 @@
+{% load i18n %}
+
+{% block subject %}
+  {% trans "Order details" context "Order confirmation email subject" %}
+{% endblock %}
+
+{% block plain %}
+{% blocktrans context "Order confirmation email text" %}
+Hi!
+
+Thank you for your order. To see your payment details please visit:
+{{ payment_url }}
+
+Sincerely,
+Saleor Team
```

Comment:
We should not hardcode "Saleor" here. Please use whatever we put in the site title.